device/vr/vr_display_impl.cc

```diff
@@ -62,13 +62,13 @@ void VRDisplayImpl::GetPassThroughCamera(const GetPassThroughCameraCallback& cal
   callback.Run(device_->GetPassThroughCamera());
 }
 
-void VRDisplayImpl::GetPlanes(const GetPlanesCallback& callback) {
+void VRDisplayImpl::GetPlaneDeltas(const GetPlaneDeltasCallback& callback) {
   if (!device_->IsAccessAllowed(this)) {
-    callback.Run(std::vector<mojom::VRPlanePtr>());
+    callback.Run(mojom::VRPlaneDeltasPtr());
```

Comment:
I am not 100% if this represents a null pointer. Same could applicable to hitTest.

Reply:
I don't fully understand your point here. What is the concern?